src/components/ShoppingListItem/ShoppingListItem.js

```diff
@@ -1,72 +1,159 @@
-          {item.status === 'inactive' && ' Inactive'}
-        </span>
-      </label>
-      <button
-        type="button"
-        aria-label={`${item.itemName} details`}
-        className="item__details-button icon-only-button"
-      >
-        <DetailsIcon aria-hidden="true" focusable="false" />
-      </button>
-      <button
-        type="button"
-        onClick={() => handleModalOpen(item)}
-        aria-controls={`item-${item.id}`} // destructive delete controls shopping list item id
-        aria-label={`Delete ${item.itemName}`}
-        className="item__delete-button icon-only-button"
+      <div className="item__primary">
+        <input
+          id={`item-input-${item.id}`}
+          value={item.id}
+          type="checkbox"
+          checked={recentlyPurchased}
+          /*remove message when no longer focused on checkbox */
+          onBlur={() => setItemNotice('')}
+          className={`checkbox item__checkbox visually-hidden ${
+            recentlyPurchased ? 'checkbox_recently-purchased' : ''
+          } item__checkbox_${item.status}`}
+          onChange={(e) =>
+            e.target.checked ? handleCheck(item) : handleUncheck(item)
+          }
+        />
+        <label
+          htmlFor={`item-input-${item.id}`}
+          className={`item__checkbox-target checkbox-target ${
+            recentlyPurchased ? 'checkbox-target_recently-purchased' : ''
+          } checkbox-target_status_${item.status}`}
+        >
+          <CheckboxIcon aria-hidden="true" focusable="false" />
+        </label>
+        <label
+          className={`label label_check-radio item__label item__label_${item.status}`}
+          htmlFor={`item-input-${item.id}`}
+        >
+          {item.itemName}
+          <span className="visually-hidden">
+            {/* text for screen readers only, based on item.status set in ShoppingList with getItemStatus function */}
+            {item.status === 'soon' && ' Need to buy soon'}
+            {item.status === 'kind-of-soon' && ' Need to buy kind of soon'}
+            {item.status === 'not-soon' && " Don't need to buy soon"}
+            {item.status === 'inactive' && ' Inactive'}
+          </span>
+        </label>
+        <button
+          type="button"
+          aria-label={`${item.itemName} details`}
+          className="item__details-button icon-only-button"
+        >
+          <DetailsIcon aria-hidden="true" focusable="false" />
+        </button>
+        <button
+          type="button"
+          onClick={() => handleModalOpen(item)}
+          aria-controls={`item-${item.id}`} // destructive delete controls shopping list item id
+          aria-label={`Delete ${item.itemName}`}
+          className="item__delete-button icon-only-button"
+        >
+          <DeleteIcon aria-hidden="true" focusable="false" />
+        </button>
+      </div>
+
+      {/* A space for item-related user feedback, which may be visual or for screen readers only */}
+      <div
```

Comment:
Since you aren't actually holding space here should this be a conditional render instead of an empty div when no message is present?

Reply:
@abowler2 I tested that with screen readers and if the aria-live is not present beforehand, the message doesn't seem to get read! Seems like there needs to be an empty div there that the screen reader knows to watch?

Reply:
Hmmm.... I didn't experience that but then I'm only testing with VoiceOver so it may be different for other screen readers.  Since there is no logic within that div, it's not a big deal to leave it as is.

Reply:
Just tried it again to double check and yeah, NVDA and Jaws don't like it. 🙄